python/tests/integration/connectors/memory/test_azure_cosmosdb_memory_store.py

```diff
@@ -103,8 +108,8 @@ async def test_create_get_drop_exists_collection():
 
     await store.delete_collection(test_collection)
 
-    result = await store.does_collection_exist(test_collection)
-    assert result is True
+    result = await store.does_collection_exist_async(test_collection)
```

Comment:
We've removed all *_async suffixes from method names, if the only method being used is async. This method should be `does_collection_exist()`.

Reply:
Please also run `black` formatting.